convert.py

```diff
@@ -29,10 +30,20 @@
     default="", 
     help="The filepath of the .epub file to convert"
 )
+parser.add_argument("--offline", 
+    action='store_true',
+    help="Whether to use the offline model for parsing. If this flag is passed, offline model is used"
+)
 args = parser.parse_args()
 
 # The filepath of the eBook to convert.
 EPUB_FILEPATH = args.file
+ENABLE_OFFLINE = args.offline
+if not ENABLE_OFFLINE:

```

Comment:
Love the dynamic imports! Personally I'd suggest:

if ENABLE_OFFLINE:
    from tts_utils import TTS
    import re
else:
   from google.cloud import texttospeech

to avoid the double negative (just fewer mental gymnastics, since we have an ```if not``` and an else)

Reply:
ack